src/order/lattice.lean

```diff
@@ -646,6 +646,26 @@ by apply_instance
 -/
 namespace monotone
 
+/-- Pointwise supremum of two monotone functions is a monotone function. -/
+protected lemma sup [preorder α] [semilattice_sup β] {f g : α → β} (hf : monotone f)
+  (hg : monotone g) : monotone (λ x, f x ⊔ g x) :=
```

Comment:
Possibly better as
```suggestion
  (hg : monotone g) : monotone (f ⊔ g) :=
```

Reply:
I don't want to touch too much code, and `pi.lattice` is in `order.bounded_lattice` now. Feel free to move it to `order.lattice` in another PR.

Reply:
Done in #9305.